docs/mkdocs.yml

```diff
@@ -31,6 +31,8 @@ nav:
     - performance.md
     - reliability.md
     - schemas.md
+  - Views:
```

Comment:
We might want to rethink the structure at some point, but this is good for now. A lot of the stuff under Tables is also applicable for Views.